openedx/features/wikimedia_features/admin_dashboard/settings/common.py

```diff
@@ -0,0 +1,11 @@
+
+"""Common settings for Admin Dashboard"""
```

Comment:
To keep the things `consistent`, kindly add docstrings in other newly added files as well.
```
"""
My docstring here.
"""
And follow this pattern